src/navability/entities/Variable.jl

```diff
@@ -3,9 +3,11 @@ using JSON
 
 DFG_VERSION = "0.18.1";
 
+#TODO: Is this used? If not, we should remove.
```

Comment:
I would like to use it, but enums are weird and don't work right. This is the root of all those convos with Johan about enums. Maybe open an issue to implement enums.

Reply:
I've fixed it with a dictionary that does the same functional thing. Strings give too much room for error.